test/spawn.jl

```diff
@@ -64,12 +64,12 @@ if valgrind_off
 end
 
 if is_unix()
-    prefixer(prefix, sleep) = `perl -nle '$|=1; print "'$prefix' ", $_; sleep '$sleep';'`
-    @test success(pipeline(`perl -le '$|=1; for(0..2){ print; sleep 1 }'`,
-                       prefixer("A",2) & prefixer("B",2)))
-    @test success(pipeline(`perl -le '$|=1; for(0..2){ print; sleep 1 }'`,
-                       prefixer("X",3) & prefixer("Y",3) & prefixer("Z",3),
-                       prefixer("A",2) & prefixer("B",2)))
+    prefixer(prefix, sleep) = `sh -c "while IFS= read REPLY; do echo '$prefix ' \$REPLY; sleep $sleep; done"`
```

Comment:
Busybox is porported to be ash. Sample code online shows this should be valid. I'm not sure how it would be done more portably (the sleep call support is similar). I think this is at least as general as the previous perl solution (which also isn't present in busybox)

If we want something really generic, the c script is just as short. I can write that if you think we should do that instead.